mscore/editlyrics.cpp

```diff
@@ -30,6 +31,8 @@ bool ScoreView::editKeyLyrics()
       {
       Q_ASSERT(editData.element->isLyrics());
 
+      void insertSym(EditData& ed, SymId id);
```

Comment:
Adding the prototype here doesn't make a protected member visible to a foreign class